packages/esm-patient-orders-app/src/hooks/useIsBackendModuleInstalled.tsx

```diff
@@ -0,0 +1,27 @@
+import { type FetchResponse, openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+import useSWR from 'swr';
+import { useMemo } from 'react';
+
+interface ModuleData {
+  results: Array<{ uuid: string }>;
+}
+
+export const useIsBackendModuleInstalled = (modules: string | string[]) => {
```

Comment:
```suggestion
export const useAreBackendModulesInstalled = (modules: string | string[]) => {
```

Because we're dealing with a list of modules